developers/node-tutorial.md

```diff
@@ -237,9 +237,13 @@ commands:
 3. Build the `celestia` binary:
 
    ```bash
-   make build
+   make build-jemalloc
```

Comment:
We should keep the regular one and add jemalloc as a secondary option.

Reply:
sounds good, sorry, was going off of [this](https://github.com/celestiaorg/docs/issues/1308#issuecomment-1852434735)

Reply:
Miscomunice. I thought of this as a secondary option for all the node types

Reply:
we good! just made PR :smile: